vllm/model_executor/models/jamba.py

```diff
@@ -364,6 +438,8 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
         assert not cache_config.enable_prefix_caching, \
             "Jamba currently does not support prefix caching"
 
+        self.quant_config = vllm_config.quant_config
```

Comment:
On line 421, there is the `supported_lora_modules` that can be expanded to include all of the target_modules listed when running tuning with "all-linear". This would include:
```
  "target_modules": [
    "down_proj",
    "v_proj",
    "up_proj",
    "q_proj",
    "k_proj",
    "in_proj",
    "o_proj",
    "out_proj",
    "gate_proj"
  ],
```